CodeEdit/Features/Acknowledgements/Views/AcknowledgementsView.swift

```diff
@@ -8,44 +8,43 @@
 import SwiftUI
 
 struct AcknowledgementsView: View {
-
-    @Environment(\.openURL) private var openURL
-
     @ObservedObject
     private var model = AcknowledgementsViewModel()
 
+    @State private var displayDivider = false
+
     var body: some View {
-        VStack {
+        VStack(spacing: 0) {
             Image(nsImage: NSApp.applicationIconImage)
                 .resizable()
-                .frame(width: 70, height: 70)
+                .frame(width: 48, height: 48)
             Text("Acknowledgements")
-                .font(.largeTitle)
+                .font(.title)
                 .fontWeight(.bold)
-            ScrollView {
-                ForEach(model.acknowledgements, id: \.name) { acknowledgement in
-                    HStack {
-                        Text(acknowledgement.name)
-                            .font(.title3)
-                            .fontWeight(.semibold)
-
-                        Spacer()
-
-                        Button {
-                            openURL(acknowledgement.repositoryURL)
-                        } label: {
-                            Image(systemName: "arrow.right.circle.fill")
-                                .foregroundColor(Color(nsColor: .tertiaryLabelColor))
+                .padding(.vertical, 8)
+            Divider()
+                .opacity(displayDivider ? 1 : 0)
+            OffsettableScrollView(showsIndicator: false) { offset in
+                displayDivider = offset.y < 0
+            } content: {
+                LazyVStack(spacing: 0) {
+                    ForEach(
+                        Array(zip(model.acknowledgements.indices, model.acknowledgements)),
```

Comment:
Please move this to the view model as a computed property like so:

```swift
var indexedAcknowledgements: [(index: Int, acknowledgement: AcknowledgementDependency)] {
  return Array(zip(acknowledgements.indices, acknowledgements))
}
```

Reply:
Like that?

Reply:
Yeah and now just call it like this:

```swift
ForEach(model.indexedAcknowledgements, id: \.1.name) { … }

// or

ForEach(model.indexedAcknowledgements, id: \.acknowledgement.name) { … }
```